### PR TITLE
Hotfixes for Units Manager and Mods Manager

### PR DESCRIPTION
**Pull request by HUSSAR-mtrela**
@aeoncleanse as you reqeusted I made separate commits and this PR fixes for #1174 and other minor issues in UI for UnitsManger and ModsManager (#401)

### CODE REVIEW

File: lua/ui/lobby/UnitsAnalyzer.lua [hunk 1]
```diff
@@ -545,10 +562,12 @@ function GetWeaponsStats(bp)
...
 
     for id, w in bp.Weapon or {} do
-            
+        local damage = w.NukeInnerRingDamage or w.Damage
+        -- skipping not important weapons  
         if w.WeaponCategory and 
            w.WeaponCategory ~= 'Death' and 
-           w.WeaponCategory ~= 'Teleport'then
+           w.WeaponCategory ~= 'Teleport' and
+           damage > 0 then
```
Comment (aeoncleanse) [inline lua/ui/lobby/UnitsAnalyzer.lua]: Why are we now checking damage in a comments commit?

File: lua/ui/lobby/UnitsAnalyzer.lua [hunk 2]
```diff
@@ -547,10 +562,12 @@ function GetWeaponsStats(bp)
     --check bp.EnhancementPresetAssigned.Enhancements table to get accurate stats
 
     for id, w in bp.Weapon or {} do
-            
+        local damage = w.NukeInnerRingDamage or w.Damage
```
Comment (aeoncleanse) [inline lua/ui/lobby/UnitsAnalyzer.lua]: Should be inner + outer. They stack.

File: lua/ui/lobby/UnitsAnalyzer.lua (hunk 1 above)
Comment (HUSSAR-mtrela) [inline lua/ui/lobby/UnitsAnalyzer.lua]: UEF shield boat has fake weapon to show range of shield or something like that so this ensure we dont show fake weapons

File: lua/ui/lobby/UnitsAnalyzer.lua (hunk 2 above)
Comment (HUSSAR-mtrela) [inline lua/ui/lobby/UnitsAnalyzer.lua]: did not know that and I will correct it soon

**Comment by HUSSAR-mtrela** [comment]
My commit 7c44597 fixes issue #401

**Comment by speed2CZ** [comment]
I've tried it, seems working, haven't found any problem...

**Comment by aeoncleanse** [comment]
Merged b3c8b52da8b7545ad467c462c23809d5f5027a6c

**Pull request closed without merging**